README.md

```diff
@@ -27,13 +27,29 @@ compatibility. When possible, upgrade your Conjur version to match the
 when using integrations, choose the latest suite release that matches your Conjur version. For any 
 questions, please contact us on [Discourse](https://discuss.cyberarkcommons.org/c/conjur/5).
 
-### From PyPI
+## Supported Platforms
+
+- Red Hat Enterprise Linux (7, 8)
+- Windows
+- macOS
+
+### About our releases
+
+#### Github releases
+
+GitHub releases are created for GitHub tagged versions that have undergone additional quality testing. With each release, we push the new release archived files for each of our supported platforms to [our release page](https://github.com/cyberark/conjur-api-python3/releases).
```

Comment:
@sgnn7 the releases here are for both the API and the CLI correct?

Reply:
Correct. Zip is the API (client), and binaries are the CLI

Reply:
@sgnn7 the only zip in the releases is the source code. Is that what you mean?

Reply:
yup